services/workflows-service/src/providers/file/file.service.ts

```diff
@@ -219,6 +222,16 @@ export class FileService {
       };
     }
 
+    if (
+      provider == 'base64' &&
+      z.string().refine(Base64.isValid).parse(extractBase64Payload(uri))
```

Comment:
Should `safeParse` be used here or do we want to throw a validation error on this? See if using the `ValidationError` class makes sense here.

Reply:
P1

Reply:
It should fail, we cant process base64 when value is not.